lib/commands/fastboot.js

```diff
@@ -15,27 +16,29 @@ module.exports = {
     { name: 'assets-path', type: String, default: 'dist' }
   ],
 
-  runCommand: function(appName, options) {
-    var commandOptions = this.commandOptions;
-    var outputPath = commandOptions.outputPath;
-    var assetsPath = commandOptions.assetsPath;
+  startServer: function(options) {
     var ui = this.ui;
+    var self = this;
 
-    ui.writeLine("Installing FastBoot npm dependencies");
+    if (this._fastBootListener) {
```

Comment:
Ah, now I remember where this was an issue: https://github.com/ember-cli/ember-cli/blob/da4749d3b27a2ce7c529de0d07c0ac3e702e7d47/lib/tasks/server/express-server.js#L238-L250